prometheus/doc/alert-manager.md

```diff
@@ -58,3 +58,28 @@ cd $pai-management
 ```
 
 Please fire a pull request if you find any rule useful.
+
+# Muting firing alert
+
+OpenPAI leverage [amtool](https://github.com/prometheus/alertmanager#amtool) to interactive with
+alert manager. If you're using [dev-box](../../pai-management/doc/cluster-bootup.md#c-step-0) to do
+cluster management, the amtool might has already been installed for you, you can try to execute
+`amtool` in your shell to test. If it hasn't been installed, please install it using method provided
+by amtool [README](https://github.com/prometheus/alertmanager#install-1).
+
+To ease configuration, OpenPAI provides a script that generate amtool config file for you, you can use
+
+```
+python utilities/gen-amtool-config.py -p /cluster-configuration
```

Comment:
will this trigger muting ? or just reconfig? if trigger, please tell user after execute will mute

Reply:
this only generate configuration for amtool, user should call amtool for muting or something.

Reply:
I mentioned that [here](https://github.com/Microsoft/pai/pull/1261/files/6e754af8b9c788256a740872feb3f101e6276908#diff-f27c2b85c290aaf4f5f812da80fad884R84)

Reply:
for me first time read this, not aware and have the question above.

copy this (or tell user could find Silence an alert cmd at that file will be intuitive): 
Silence an alert

$ amtool silence add alertname=Test_Alert
b3ede22e-ca14-4aa0-932c-ca2f3445f926

$ amtool silence add alertname="Test_Alert" instance=~".+0"
e48cb58a-0b17-49ba-b734-3585139b1d25

Reply:
updaed